### DISCUSSION TITLE
change the opacity of theme color

### DISCUSSION THREAD

**Discussion by ShaifArfan** [mantinedev/mantine#3934] [Q&A] [answered]
Is there any way to change the opacity of theme color?

**Comment by rtivital** [comment, marked as answer]
You can use `theme.fn.rgba` function for that, for example, `theme.fn.rgba(theme.colors.red[7], 0.5)`. See docs – https://mantine.dev/theming/functions/#rgba

**Reply by ShaifArfan** [reply]
Thank You. I got it.
